src/InformaticsGateway/Test/Services/Connectors/PayloadMoveActionHandlerTest.cs

```diff
@@ -190,25 +190,31 @@ public async Task GivenAPayload_WhenAllFilesAreMove_ExpectPayloadToBeAddedToNoti
             });
 
             var correlationId = Guid.NewGuid();
+            var file1 = new DicomFileStorageMetadata(correlationId.ToString(), Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), Guid.NewGuid().ToString());
```

Comment:
would it be worth declaring a ctor that can take GUID?

example
![image](https://github.com/Project-MONAI/monai-deploy-informatics-gateway/assets/61380713/9c1b2dd1-6a2e-4801-b754-3f2cba737b02)


`        
public DicomFileStorageMetadata(Guid associationId, Guid identifier, Guid studyInstanceUid, Guid seriesInstanceUid, Guid sopInstanceUid) : this(
            associationId.ToString(),
            identifier.ToString(),
            studyInstanceUid.ToString(),
            seriesInstanceUid.ToString(),
            sopInstanceUid.ToString())
        { }
`

Reply:
looking at the code the only benefit for the above change would be the tests. so ....